chroot-bin/croutonfindnacl

```diff
@@ -52,7 +52,10 @@ for pid in $PIDS; do
 
     # Iterate over mappings, and check signature
     for fd in "/proc/$pid/fd"/*; do
+	set +e
         link="`readlink -- "$fd"`"
```

Comment:
I think you can just add || true and the if statement below will naturally fall through.

`link="$(readlink -- "$fd" || true)"`